packages/data-point/lib/reducer-types/reducer-list/factory.test.js

```diff
@@ -86,6 +98,16 @@ describe('factory#create', () => {
     expect(result.reducers[2].type).toBe('ReducerPath')
   })
 
+  test('factory#create reducer from grouped reducers with multiple spaces', () => {
+    const result = factory.create(createReducer, [
+      '$foo    |    $bar    | $baz    '
+    ])
+    expect(result.reducers).toHaveLength(3)
+    expect(result.reducers[0].type).toBe('ReducerPath')
+    expect(result.reducers[1].type).toBe('ReducerPath')
+    expect(result.reducers[2].type).toBe('ReducerPath')
+  })
```

Comment:
great tests - my only suggestion is that maybe we should have leading whitespace in one of the input strings to make sure it's trimmed

Reply:
Good point. Just pushed an update with that suggestion.

Reply:
so fast!

Reply:
Already finished my sprint. 🤣